autosklearn/pipeline/components/data_preprocessing/imputation/numerical_imputation.py

```diff
@@ -5,8 +5,19 @@
 from autosklearn.pipeline.constants import *
 
 
-class Imputation(AutoSklearnPreprocessingAlgorithm):
-    def __init__(self, strategy='median', random_state=None):
+class NumericalImputation(AutoSklearnPreprocessingAlgorithm):
+    """ Imputation of numerical features. By default, replace missing values by the 
+    mean of the others.
+
+    Parameters
```

Comment:
Hey, you don't need to add a docstring for models which are simply a wrapper of a scikit-learn class.

Reply:
ok